src/std/sugar-test.ss

```diff
@@ -58,4 +59,16 @@
     (check ((is 'a test: eq?) 'a)       => #t)
     (check ((is 2.0) 2.0)               => #t)
     (check ((is "a") "a")               => #t))
+
+   (test-case "test assert! failure message"
+    (def e 'needle)
+    (def l ['stack 'of 'hay])
+    (check-exception (assert! (member e l))
+                     (lambda (e)
+                       (regexp-match?
+                        (string-append
+                         "Assertion failed \"sugar-test.ss\"@66.31: \\(member e l\\)\n"
```

Comment:
lets fuzzy match the line/column.

Reply:
Allow anything digits for the line, but match the strict column?
Or something like match the tens place for the column while allowing any digit for the ones place?

Reply:
digits for line, strict column should be fine.

Reply:
Okay, I've changed it to use pregexps and `\"sugar-test.ss\"@\\d+\\.31` to allow any digits for the line.